src/Api/MailingList/Member.php

```diff
@@ -153,6 +156,7 @@ public function createMultiple(string $list, array $members, $upsert = false)
                     case 'vars':
                         if (is_array($value)) {
                             $value = json_encode($value);
+                            $parameters[$field] = $value;
```

Comment:
i hope, `$parameters[$field] = $value;` should be `$members[$field] = $value`;

Reply:
Line removed as not wrong and not needed anymore. Because passing by reference is using here. Passing by value in `update` function was replaced